src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -179,7 +179,9 @@ void BasePlaylistFeature::selectPlaylistInSidebar(int playlistId, bool select) {
 void BasePlaylistFeature::activateChild(const QModelIndex& index) {
     //qDebug() << "BasePlaylistFeature::activateChild()" << index;
     int playlistId = playlistIdFromIndex(index);
-    VERIFY_OR_DEBUG_ASSERT(playlistId != kInvalidPlaylistId) {
+    if (playlistId == kInvalidPlaylistId) {
```

Comment:
Please check that the comment is valid with `DEBUG_ASSERT(hasChildren(index))`. We don't expect empty inner nodes without any children.

Reply:
I think this can happen if you delete the last history play list from a year folder.

Just tested and it crashes for another reason. I will have a look.